examples/pennylane/0_Getting_started/0_Getting_started.ipynb

```diff
@@ -332,7 +332,7 @@
     },
```

Comment:
<p>We should consider gating execution of this notebook due to the possibility of incurred costs</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/683/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>

Reply:
Yes, I'm fixing it by uncommenting the Ankaa-3 device in the code block, and replacing it with SV1.